packages/mirage-flow-lwt/mirage-flow-lwt.1.2.0/opam

```diff
@@ -17,7 +17,8 @@ depends: [
   "topkg" {build & >= "0.7.3"}
   "fmt"
   "lwt"
-  "cstruct" {>= "2.0.0"}
+  ("cstruct" {>= "2.0.0" & < "3.0.0"}
+  | ("cstruct" {>= "3.0.0"} & "cstruct-lwt" {>= "3.0.0"}))
```

Comment:
i tried to explain this further in https://discuss.ocaml.org/t/ann-cstruct-3-0-0-with-packaging-changes/284/4